Godeps/Godeps.json

```diff
@@ -896,6 +909,10 @@
 			"Rev": "c2528b2dd8352441850638a8bb678c2ad056fd3e"
 		},
 		{
+			"ImportPath": "golang.org/x/net/http2",
```

Comment:
It *looks* like the only thing that depended on bradfitz/http2 was the old version of grpc.  If the new version of GRPC is depending on net/http2, I think we no longer need the old version.  Can you check the dependency tree locally?